stl/inc/concepts

```diff
@@ -157,7 +157,7 @@ namespace ranges {
             template <class _Ty1, class _Ty2, size_t _Size>
             constexpr void operator()(_Ty1 (&__t)[_Size], _Ty2 (&__u)[_Size]) const
                 noexcept(noexcept((*this)(__t[0], __u[0])))
-                requires requires(const _Cpo& __f) { __f(__t[0], __u[0]); } { // TRANSITION, LLVM-44689
+                requires requires { (*this)(__t[0], __u[0]); } {

```

Comment:
To confirm my understanding: it's okay to remove this workaround before increasing the required Clang version in yvals_core.h because Clang 9 inherently doesn't support `<concepts>`, so removing a workaround within it doesn't matter?

Reply:
Yes, that's the case. This is a workaround for a clang-10-rc1 bug that was fixed in a later release candidate, so the workaround isn't needed by any released compiler.